plugin.video.fox.news/resources/lib/scraper.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+# Fox News Kodi Video Addon
+#
+from t1mlib import t1mAddon
+import json
+import re
+import os
+import urllib
+import xbmc
+import xbmcplugin
+import xbmcgui
+import HTMLParser
+import sys
+
+h = HTMLParser.HTMLParser()
+qp  = urllib.quote_plus
+uqp = urllib.unquote_plus
+UTF8 = 'utf-8'
+
+
+class myAddon(t1mAddon):
+
+  def getAddonMenu(self,url,ilist):
+      ilist = self.addMenuItem('Live','GV', ilist, '5614615980001', self.addonIcon, self.addonFanart, {}, isFolder=False)
+      ilist = self.addMenuItem('Show Clips','GM', ilist, 'show', self.addonIcon, self.addonFanart, {}, isFolder=True)
+      ilist = self.addMenuItem('News Clips','GM', ilist, 'news', self.addonIcon, self.addonFanart, {}, isFolder=True)
+      return(ilist)
+
+
+
+  def getAddonMovies(self,url,ilist):
+      html = self.getRequest('https://video.foxnews.com')
```

Comment:
probably move the url to a common variable? It's used in more than one method

Reply:
done